dotnet/src/webdriver/DevTools/DevToolsDomains.cs

```diff
@@ -33,14 +34,23 @@ public abstract class DevToolsDomains
         // This is the list of known supported DevTools version implementation.
         // When new versions are implemented for support, new types must be
         // added to this dictionary.
-        private static readonly Dictionary<int, Type> SupportedDevToolsVersions = new Dictionary<int, Type>()
+        private static readonly Dictionary<int, DomainType> SupportedDevToolsVersions = new Dictionary<int, DomainType>()
         {
-            { 127, typeof(V127.V127Domains) },
-            { 129, typeof(V129.V129Domains) },
-            { 128, typeof(V128.V128Domains) },
-            { 85, typeof(V85.V85Domains) }
+            { 127, new DomainType(typeof(V127.V127Domains)) },
+            { 129, new DomainType(typeof(V129.V129Domains) )},
+            { 128, new DomainType(typeof(V128.V128Domains) )},
+            { 85, new DomainType(typeof(V85.V85Domains)) },
         };
 
+        /// <summary>Workaround for trimming.</summary>
```

Comment:
This is the kind of workaround necessary when using a `Type` as a generic argument, which requires trimming annotations. Hope this is acceptable.